doc/kubernetes/modules/ROOT/pages/running/metrics/jgrp_metrics.adoc

```diff
@@ -0,0 +1,198 @@
+Unicast messages are one-to-one messages while multicast are one-to-many messages.
+{jdgserver_name} mostly uses unicast messages.
+
+TIP: A healthy cluster shows a value of zero for all metrics.
+
+|===
+|Metric |Description
+
+| `vendor_jgroups_ufc_get_number_of_blockings`
+| The number of times flow control blocks sender for unicast messages.
+
+| `vendor_jgroups_ufc_get_average_time_blocked`
+| Average time blocked (in ms) in flow control when trying to send an unicast message.
+
+| `vendor_jgroups_mfc_get_number_of_blockings`
+| The number of times flow control blocks sender for multicast messages.
+
+| `vendor_jgroups_mfc_get_average_time_blocked`
+| Average time blocked (in ms) in flow control when trying to send a multicast message.
+
+|===
+
+== Retransmissions
+
+JGroups provides reliable delivery of messages.
+When a message is dropped, on the network or the receiver cannot handle the message, a retransmission is required.
+Retransmissions increase resources usage, and it is usually a signal of an overload system.
+
+Random Early Drop (RED) monitors the sender queues.
+When the queues are almost full, the message is dropped, and a retransmission must happen.
+It prevents threads from being blocked by a full sender queue.
+
+TIP: A healthy cluster shows a value of zero for all metrics.
+
+|===
+|Metric |Description
+
+| `vendor_jgroups_unicast3_get_num_xmits`
+| The number of retransmitted messages.
+
+| `vendor_jgroups_red_get_dropped_messages`
+| The total number of dropped message by the sender.
+
+| `vendor_jgroups_red_get_drop_rate`
+| Percentage of all messages that were dropped by the sender.
+
+|===
+
+== Network Partitions
+
+=== Cluster Size
+
+The cluster size metric reports the number of nodes present in the cluster.
+In a healthy cluster, all nodes must report the same number.
+If it differs, it may signal that a node is joining, shutdown or, in the worst case, a network partition is happening.
+
+TIP: A healthy cluster shows the same value in all nodes.
+
+|===
+|Metric |Description
+
+| `vendor_cluster_size`
+| The number of node in the cluster.
+
+|===
+
+=== Cross-Site Status
+
+The cross-site status reports connection status to the other site.
+It returns a value of `1` if it is connection or `0` if disconnected.
+The value of `2` is used on nodes where the status is unknown; not all node establish connections to the remote sites.
+
+TIP: A healthy cluster shows a value greater than zero.
+
+|===
+|Metric |Description
+
+| `vendor_jgroups_site_view_status`
+| The single site status (1 if online).
+
+|===
+
+====
+*Tags*
+
+* `site=<name>`: The name of the destination site.
+====
+
+=== Network Partition Events
+
+Network partitions in a cluster can happen due to various reason.
+This metrics does not help predict network splits but signals that it happened, and the cluster has been merged.
+
+TIP: A healthy cluster shows a value of zero for this metric.
```

Comment:
I'd love to see more `TIP: A healthy cluster...` mentions 😍 - still it could be a future iteration.